lib/ts/usercontext/userContextThemeWrapper.tsx

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+import React from "react";
+import { UserContextContext, UserContextProvider } from ".";
+
+export default function UserContextWrapper(props: { children: React.ReactNode; userContext?: any }): JSX.Element {
+    /**
```

Comment:
Whatever userContext object you are passing down, you need to observe it for changes. Any changes should cause a rerender.

To observe changes, see https://medium.com/@suvechhyabanerjee92/watch-an-object-for-changes-in-vanilla-javascript-a5f1322a4ca5

Reply:
@nkshah2 this comment was missed.

Reply:
As discussed we will not be adding a way to observe changes to userContext